bert_embeddings/bert_embeddings.ipynb

```diff
@@ -8,85 +8,32 @@
     "This notebook presents deployment of pretrained BERT model that outputs embeddings for given textual sequences as a serverless function. Embeddings are meaningful, contextual representations of text in the form of ndarrays that are used frequently as input to various learning tasks in the field of NLP."
```

Comment:
<p>Please add comments that explain that this is mocked:</p><p> </p><p>before line 1:</p><p>\# Mocked Nuclio context</p><p> </p><p>before line 3:</p><p>\# Mocked conext initialization</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>